package.json

```diff
@@ -30,9 +30,11 @@
   ],
   "dependencies": {
     "carbon-icons": "^6.0.4",
+    "flatpickr": "^2.6.3",
     "lodash.debounce": "^4.0.8"
   },
   "devDependencies": {
+    "@ibma/karma-ibma": "^2.2.0",
```

Comment:
@hellobrian You are right that we have removed karma-ibma dependency from package.json, as [it has to be downloaded from https://aat.mybluemix.net/dist/karma-ibma.tgz](https://github.com/carbon-design-system/carbon-components/blob/0a9da53/.travis.yml#L42) (as it’s an internal dependency).